addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -338,10 +338,12 @@
 <template id='course_card' name="Course Card">
     <div t-attf-class="card w-100 o_wslides_course_card mb-4 #{'o_wslides_course_unpublished' if not channel.is_published else ''}" t-cache="channel if is_public_user and not search_tags else None">
         <t t-set="channel_frontend_tags" t-value="channel.tag_ids.filtered(lambda tag: tag.color)"/>
-        <a t-attf-href="/slides/#{slug(channel)}" t-title="channel.name" style="height:120px">
+        <a class="position-relative" t-attf-href="/slides/#{slug(channel)}" t-title="channel.name" style="height:120px">
```

Comment:
Not necessary here, its parent already has the `position: relative;` rule

Reply:
Hi @pko-odoo :slightly_smiling_face: 
Updated! Nice catch!